src/WordPress.php

```diff
@@ -176,6 +176,9 @@ public function __construct() {
 	 *
 	 * [--ssl=<value>]
```

Comment:
Remove `=<value>`, so that passing value is not necessary.

Reply:
Do we have other option besides `le` for ssl?